deployments/deploy-ssh.sh

```diff
@@ -9,9 +9,14 @@ if [ -z "${ENABLE_DASH}" ]; then
 fi
 
 if [ -z "${DASH_UID}" ]; then
-    DASH_UID="0"
+    if [ -e /build.properties ]; then
+      DASH_UID=`cat /build.properties | grep -oE '[0-9]+'`
```

Comment:
This logic will break if there are multiple env variables in the build.prop.. recently I added a logic for this in spawn.groovy. Take a look at that